### PR TITLE
Fixes for issues #2741, #2944, #2955

### LINKED ISSUE #2741: User experience: confusing message after making a payment
When I made a payment for my trade, a message appeared that was a little confusing to me.

I later learned that it means that the trader wasn’t online and will receive the message as soon she gets online again.

Suggestion: Rephrase the message to something like this:  
"Message of payment sent but not yet confirmed by peer."

Confusing message

### LINKED ISSUE #2944: High/low values are swapped in trade statistics candle popup
While viewing the trade statistics for BSQ, I hovered over the monthly candle and the popup showed the high/low values swapped.
image

### LINKED ISSUE #2955: Show median value in trade statistics candle popup
The trade statistics candle popup currently shows the average, but perhaps should also show the median.

See the discussion in https://github.com/bisq-network/bisq/pull/2937 for more details. But to summarize, showing the median value would be beneficial for determining BSQ price for compensation request purposes. Using the median as opposed to average would ensure outlier trades would have a minimal influence on the determined price.

image

### PR DESCRIPTION
**Pull request by niyid**
1. Issue #2944 is a bug. The low and high price values were inverted when the currency type is cryptocurrency.

2. Issue #2955 is a feature add - inclusion of median statistic for display in the candle tool-tip.

3. Issue #2741 is a simple change in i18n message bundle for better clarity.

fixes #2741, fixes #2955, fixes #2944

### CODE REVIEW

**Comment by niyid** [comment]
Hi Christopher,

I think I finally got it right this time. Sorry for the inconvenience and thanks for your patience.

Regards.

**Comment by niyid** [comment]
Should median computation function be added to **`bisq.common.util.MathUtils`** rather than locally to **`bisq.desktop.main.market.trades.TradesChartsViewModel`** as **`Long findMedian(Long[] prices)`**?

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -302,6 +302,7 @@ market.trades.tooltip.candle.close=Close:
 market.trades.tooltip.candle.high=High:
 market.trades.tooltip.candle.low=Low:
 market.trades.tooltip.candle.average=Average:
+market.trades.tooltip.candle.average=Median:
```
Comment (ripcurlx) [inline core/src/main/resources/i18n/displayStrings.properties]: You used the same identifier as for average

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - please see my translation comment

File: core/src/main/resources/i18n/displayStrings.properties (hunk 1 above)
Comment (niyid) [reply core/src/main/resources/i18n/displayStrings.properties]: That is now corrected. Apologies.

**Review by ripcurlx — APPROVED** [review]

**Comment by ripcurlx** [comment]
@niyid For your next PRs please make one PR per issue, so the PRs are as small as possible. Thanks!

**Comment by niyid** [comment]
Will do. Apologies for any mix-up,.

**Review by niyid — COMMENTED** [review]
Hi @ripcurlx. Can you please reject this latest commit to that I can push on a separate PR? Apologies.

In the interim I have renamed the pull request in order to be able to make sense out of it.

**Comment by ripcurlx** [comment]
> Hi @ripcurlx. Can you please reject this latest commit to that I can push on a separate PR? Apologies.

@niyid Do you mean I should close this PR? I don't know what you mean with rejecting a commit.

**Comment by niyid** [comment]
> > Hi @ripcurlx. Can you please reject this latest commit to that I can push on a separate PR? Apologies.
> 
> @niyid Do you mean I should close this PR? I don't know what you mean with rejecting a commit.

Sorry about the choice of words. I had mistakenly pushed a commit for #1896 on this same PR. I then created a separate PR for #1896 after realizing this mistake.

**Comment by ripcurlx** [comment]
@niyid I just wanted to test the PR but saw that it has a a conflict in Namecoin.java. As soon as this is fixed, I'll test the PR.

**Comment by ripcurlx** [comment]
Also the tests for the coins are failing for me. Somehow Travis didn't run for this PR. I just tried to run it manually again.

**Comment by niyid** [comment]
@ripcurlx

Yes. The last commit should be deleted. Those changes (which were incomplete) are now in the completed PR for the fix to #1896.

**Comment by niyid** [comment]
I will do a revert on the last commit if that is OK with you.

**Comment by niyid** [comment]
I will see if I can revert. But the commit that should be reverted is this:
https://github.com/bisq-network/bisq/commit/363f2e97607fbc38ec45a945bdaf594469454322

**Comment by niyid** [comment]
> @ripcurlx
> 
> Yes. The last commit should be deleted. Those changes (which were incomplete) are now in the completed PR for the fix to #1896.

I have reverted the last commit. Please go ahead with the PR test. Apologies for any inconvenience.

**Review by freimair — APPROVED** [review]
Ack

**Pull request merged**